server/datastore/inmem/email_changes.go

```
@@ -0,0 +1,10 @@
+package inmem
+
+// Stubs: inmem is going away
+func (ds *Datastore) PendingEmailChange(uid uint, newEmail, token string) error {
+	return nil
```

Change stups to say `panic("deprecated")` instead of returning results if possible.